src/openrct2/peep/Staff.cpp

```diff
@@ -474,7 +476,8 @@ static uint8_t staff_handyman_direction_to_nearest_litter(Peep* peep)
     int16_t nextZ = ((peep->z + 8) & 0xFFF0) / 8;
 
     TileElement* tileElement = map_get_first_element_at(nextTile.x / 32, nextTile.y / 32);
-
+    if (tileElement == nullptr)
+        return nextDirection;
```

Comment:
`return 0xFF;` The litter is on an invalid tile in this situation so we should not try to path towards it. This might cause some really odd things to happen in almost every case though so I guess it doesn't ultimately matter what is done.